### ISSUE TITLE
Mixup between UserPreferences and Preferences in chapter 6

### ISSUE THREAD

**Issue by Tommy0304** [googlecodelabs/android-datastore#37]
class UserPreferencesRepository(
    private val userPreferencesStore: DataStore<UserPreferences>, <-- this. should be Preferences
    context: Context
) { ... }

also, hard to follow. The student does not know where to insert the new code, therefore it leads to unexpected errors

**Comment by chinalwb** [comment]
@Tommy0304 is absolutely right! The codelab is VERY confusing! It could be the WORST one I've experienced. I can hardly get the key points for using DataStore.

**Comment by mmoczkowski** [comment]
Thank you, this is noted, fix is underway. Closing in favor of #45 which has better description.
